### ISSUE TITLE
how to use the full screen listener

### ISSUE THREAD

**Issue by notacrackpot** [PierfrancescoSoffritti/android-youtube-player#610]
Hello,

I am new to coding so I am learning how to create an app. So far I managed to create an app with 3 youtube videos using the XML and shown the full-screen button. 

In non-fullscreen mode, the edges of the youtube videos are not visible but the video plays fine. Upon testing it, when I click on the full-screen button the edges become visible but it does not fill the screen of the phone.

I tried looking for fullscreen listeners online but have failed in finding one or in how to use it.

Would you know how to create a full-screen listener? I don't have any Java so far

**Comment by bazantik** [comment]
Hi, I just added an answer to a similar issue (how to use the fullscreen) [on Stack Overflow](https://stackoverflow.com/a/64144183/7475967). In a nutshell:
- create a layout for fullscreen and add it to the parent layout;
- in the .getPlayerUiController().setFullScreenButtonClickListener remove the YT player View from the layout (eg. trailerLayout) and add the same YT player View into the fullscreen layout;
- when exiting fullscreen, remove the YT player View from the fullscreen layout and add the same YT player View into the trailerLayout.

**Comment by TavernKeeper20** [comment]
> Hi, I just added an answer to a similar issue (how to use the fullscreen) [on Stack Overflow](https://stackoverflow.com/a/64144183/7475967). In a nutshell:
> 
> * create a layout for fullscreen and add it to the parent layout;
> * in the `.getPlayerUiController().setFullScreenButtonClickListener` remove the YT player View from the layout (eg. trailerLayout) and add the same YT player View into the fullscreen layout;
> * when exiting fullscreen, remove the YT player View from the fullscreen layout and add the same YT player View into the trailerLayout.
> 
> Consider it as a "fake" fullscreen" - you are rotating the whole layout while remaining in portrait mode :-)

Do you have this in Java?

**Comment by bazantik** [comment]
> Do you have this in Java?

Hi, unfortunately not, I have it only in Kotlin. But thanks to the similarities between Kotlin and Java, it should be very similar.

**Comment by TavernKeeper20** [comment]
> > Do you have this in Java?
> 
> Hi, unfortunately not, I have it only in Kotlin. But thanks to the similarities between Kotlin and Java, it should be very similar.

I've migrated my app to Kotlin just for this and the code seems to not work for some reason. Can I contact you privately? I've been struggling with this for days and would appreciate the help.

**Comment by BuzzAntique** [comment]
Yes, of course, feel free to contact me via e-mail (in my profile) - although I am not a skilled developer, I am just learning how to code :-)

**Comment by TavernKeeper20** [comment]
Sent an email
